packages/data-broker/src/DataBroker.js

```diff
@@ -113,7 +113,10 @@ export class DataBroker {
     if (countDistinct(dataSources, 'service_type') > 1) {
       throw new Error('Cannot pull metadata for data sources belonging to different services');
     }
-    const service = this.createService(dataSources[0].service_type);
+    const serviceType = dataSources[0].service_type;
+    // `indicator` service is not support `.pullMetadata()`
+    if (serviceType === 'indicator') return [{ code: dataSources[0].code, name: undefined }];
```

Comment:
I think if we are going to do this, we might as well just implement `pullMetadata()` for indicators, no?

I think what you've got here should suffice for now, so if you just edit `pullMetadata()` in `data-broker/src/services/indicator/IndicatorService.js` to be:
```
  async pullMetadata(dataSources) {
    return dataSources.map(dataSource => ({ code: dataSource.code, name: undefined }));
  }
```

Ultimately we can properly support this interface just by adding a `name` field to the indicators table, and then fetching the `code` and `name` when `pullMetadata()` is called. But I rekon that can be done in separate ticket (I've raised https://github.com/beyondessential/tupaia-backlog/issues/2137 for this)